cron/controller/Dockerfile

```diff
@@ -26,6 +26,8 @@ ARG TARGETARCH
 RUN CGO_ENABLED=0 make build-pubsub
 
 FROM gcr.io/distroless/base:nonroot@sha256:bc84925113289d139a9ef2f309f0dd7ac46ea7b786f172ba9084ffdb4cbd9490
+ARG COMMIT_SHA
+ENV SCORECARD_COMMIT_SHA=${COMMIT_SHA}
```

Comment:
Why need to redeclare, can just use COMMIT_SHA ?

Reply:
I tested it without ARG, and it doesn't seem to work. I guess we need to declare it for it to accept `--build-arg` option.